Adding mechanism to change column names for display

waxlamp opened the pull request:
Closes #15

waxlamp committed:
Adding mechanism to change column names for display

curtislisle commented:
Awesome way to do this. Thanks. 

Sent from my iPhone

> On Jul 6, 2015, at 11:30 AM, Roni Choudhury <notifications@github.com> wrote:
> 
> Closes #15
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/XDATA-Year-3/EntityAlignLarge/pull/17
> 
> Commit Summary
> 
> Adding mechanism to change column names for display
> File Changes
> 
> M service/lineupdataset.py (20)
> M service/lineupdatasetdescription.py     (26)
> Patch Links:
> 
> https://github.com/XDATA-Year-3/EntityAlignLarge/pull/17.patch
> https://github.com/XDATA-Year-3/EntityAlignLarge/pull/17.diff
> —
> Reply to this email directly or view it on GitHub.
>

ronichoudhury commented:
I am going to make the "translated" column names longer and more descriptive before I merge.

waxlamp committed:
More descriptive column names

waxlamp committed:
Auto-sorting by "Combined" metric upon loading data

Also, de-emphasizing other fields besides "self reported"

Placing "self reported" in first position
